source/codegen/metadata/nifake/CHANGES.md

```diff
@@ -0,0 +1,16 @@
+# Changes in metadata from the nimi-python metadata
+
+## functions.py
+
+The following functions, not originally in nimi-python metadata, were newly added:
+- `'InitExtCal'`
+- `'CloseExtCal'`
+ 
+The following functions were tagged with `'init_method': True,` to ensure their generated service handlers register the new session
+with the session_repository.
+- `InitWithOptions`
+- `InitExtCal` : Added a 'custom_close' tag to this function, since this API has a corresponding close function called `'CloseExtCal'`
```

Comment:
We don't need to document addition of tags to `InitExtCal` and `CloseExtCal` since these are new functions introduced here anyway. Everything in these functions is a "change".

Reply:
No I think we need to document to help understand what those tags are for. @Akshata440 can you confirm?